### PR TITLE
Default columns for Freshmaker builds

### PR DESCRIPTION
**Pull request by gnaponie**
Signed-off-by: gnaponie <gnaponie@redhat.com>

### CODE REVIEW

**Comment by gnaponie** [comment]
@mprahl is some additional test needed or is it just fine?

File: src/app/tables/artifacts-table/artifacts-table.component.ts [hunk 1]
```diff
@@ -110,6 +110,9 @@ export class ArtifactsTableComponent implements OnChanges {
       case ('kojitag'):
         columns = ['builds', 'id', 'name'];
         break;
+      case ('freshmakerbuild'):
+        columns = ['id', 'name', 'original_nvr'];
```
Comment (mprahl) [inline src/app/tables/artifacts-table/artifacts-table.component.ts]: Could you also add `rebuild_nvr` please?

**Comment by mprahl** [comment]
@gnaponie could you also add a case for `freshmakerbuild` to the switch in `NodeExternalUrlPipe`? This will make the IDs links to the Freshmaker API.

**Comment by gnaponie** [comment]
@mprahl thank you for the review. Everything should be addressed now.

**Pull request merged**